cmd/cp-main.go

```diff
@@ -367,7 +365,7 @@ loop:
 	for {
 		select {
 		case <-trapCh:
-			stopParallel()
+			ctrlC = true
```

Comment:
This type of unsafe variable update is not correct @vadmeste a go-routine shouldn't read from a variable which can be edited externally. While it may work is still a racy code. I would expect a channel to be used here instead. which basically kills urlscanning.

Reply:
> This type of unsafe variable update is not correct @vadmeste a go-routine shouldn't read from a variable which can be edited externally. While it may work is still a racy code. I would expect a channel to be used here instead. which basically kills urlscanning.

You are right, fixed.